mftf/2.3/mftf-standalone.md

```diff
@@ -0,0 +1,49 @@
+---
+mftf-release: 2.3.6
+---
+
+# Setting up MFTF Standalone
+
+MFTF is a root level Magento dependency, but it can also be downloaded and used as its own standalone application to generate and run MFTF test materials.
+
+This is especially useful for development and contribution of MFTF, as it allows easy debugging and simpler git tracking of changes.
+
+## Standalone Setup
+
+Clone the MFTF repo:
+
+```bash
+$ git clone https://github.com/magento/magento2-functional-testing-framework.git
+```
+
+Run `composer install` at the MFTF root  and complete all normal setup steps using `bin/mftf` commands.
+
+File structure for standalone MFTF test files differs slightly from the Magento-embedded structure:
+
+* `.env` is located under `dev/.env`
```

Comment:
which [advanced configuration parameters](https://devdocs.magento.com/mftf/2.3/configuration.html) have to be set?